src/server/controllers/signInControllers.ts

```diff
@@ -400,13 +400,11 @@ export const oktaIdxApiSignInController = async ({
 	// get the email and password from the request body
 	const { email = '', password = '', passcode } = req.body;
 
-	const usePasscodeSignInFlag =
-		res.locals.queryParams.usePasscodeSignIn ||
-		res.locals.abTestAPI.isUserInVariant('PasscodeSignInTest', 'variant');
+	// we're using passwords if the `usePasswordSignIn` query parameter is set
+	const usePasscodeSignInFlag = !res.locals.queryParams.usePasswordSignIn;
```

Comment:
keeping the `usePasscodeSignInFlag` might be confusing now that it means that the `usePasswordSignIn`  was NOT set

Reply:
I've added comments and updated the name to make it clearer about what's going on!